product_docs/docs/pgd/5/upgrades/manual_overview.mdx

```diff
@@ -0,0 +1,224 @@
+
+1. Plan the upgrade.
+2. Prepare for the upgrade.
+3. Upgrade the server software.
+4. Check and validate the upgrade.
+
+## Upgrade planning
+
+There are broadly two ways to upgrade each node:
+
+* Upgrade nodes in place to the newer software version. See [Rolling server software upgrades](#rolling-server-software-upgrades).
+* Replace nodes with ones that have the newer version installed. See [Rolling upgrade using node join](#rolling-upgrade-using-node-join).
+
+You can use both of these approaches in a rolling manner.
+
+### Rolling upgrade considerations
+
+While the cluster is going through a rolling upgrade, mixed versions of software
+are running in the cluster. For example, suppose nodeA has PGD 3.7.16, while
+nodeB and nodeC has 4.1.0. In this state, the replication and group
+management uses the protocol and features from the oldest version (3.7.16
+in this example), so any new features provided by the newer version
+that require changes in the protocol are disabled. Once all nodes are
+upgraded to the same version, the new features are enabled.
+
+Similarly, when a cluster with WAL-decoder-enabled nodes is going through a
+rolling upgrade, WAL decoder on a higher version of PGD node produces LCRs
+with a higher pglogical version. WAL decoder on a lower version of PGD node
+produces LCRs with lower pglogical version. As a result, WAL senders on a higher
+version of PGD nodes are not expected to use LCRs due to a mismatch in protocol
+versions. On a lower version of PGD nodes, WAL senders may continue to use
+LCRs. Once all the PGD nodes are on the same PGD version, WAL senders use
+LCRs.
+
+A rolling upgrade starts with a cluster with all nodes at a prior release.
+It then proceeds by upgrading one node at a time to the newer release, until
+all nodes are at the newer release. More than two versions
+of any component must never be running at the same time. This requirement means that you must not start the new upgrade until the previous upgrade process fully finishes on all nodes.
+
+An upgrade process can take more time when 
+caution is required to reduce business risk. However, we don't recommend
+running the mixed versions of the software indefinitely.
+
+While you can use a rolling upgrade for upgrading a major version of the software,
+we don't support mixing PostgreSQL, EDB Postgres Extended, and
+EDB Postgres Advanced Server in one cluster. So you can't use this approach 
+to change the Postgres variant.
+
+!!! Warning
+    Downgrades of the EDB Postgres Distributed aren't supported. They require
+    that you manually rebuild the cluster.
+
+### Rolling server software upgrades
+
+A rolling upgrade is the process where the [server
+software upgrade](#server-software-upgrade) process is performed on each node in the
+cluster one after another while keeping the rest of the cluster
+operating.
+
+The actual procedure depends on whether the Postgres component is being
+upgraded to a new major version.
```

Comment:
Lots of passive voice here: "is performed", "is being upgraded" - could you improve this?